### ISSUE TITLE
Video Sync

### ISSUE THREAD

**Issue by tikuf** [MediaBrowser/Emby#73]
Luke can we add a couple of param to the hls command line to make sure we keep our audio and video in sync

To the video section  can we add 
-vsync vfr

and to the audio section

-af "aresample=async=1000"

This is not hls specific all command lines would benefit. 
the -vsync vfr will drop all the timestamps off the video frames so we prevent the possibility of 2 frames having the same time stamp (by default ffmpeg will duplicate or remove video frames to keep the audio in sync which can in some cases can cause a stutter)
the -af (audio filter) will resample the audio every millisecond and slightly stretch or compress it to match the video.

In other words using these we will be stretching or compressing the audio to match the video rather than ffmpegs default behavior or adding or deleting video frames to match the audio.
